zio-http/src/main/scala/zio/http/model/Headers.scala

```diff
@@ -1,7 +1,8 @@
-package zio.http
+package zio.http.model
 
 import io.netty.handler.codec.http.{DefaultHttpHeaders, HttpHeaders}
-import zio.http.headers.{HeaderConstructors, HeaderExtension, HeaderIterable}
+import zio.http.headers._
```

Comment:
we can find `headers` imported in some of the model classes. And it seems to me, that the traits in headers are mainly for data manipulation. Maybe `headers` should be a suppackage of `model`?

Reply:
I agree